src/cljs/zetawar/subs.cljs

```diff
@@ -431,11 +476,24 @@
   (when-let [[selected-q selected-r] @(selected-hex conn)]
     @(in-range-of-enemy-at? conn selected-q selected-r q r)))
 
+(deftrack friend-in-range-of-selected? [conn q r]
```

Comment:
I think it might simplify logic elsewhere to either turn this into 'repairable-friend-in-range-of-selected' (also checks if friends are repairable) or add that as a second track.